docs/source/quick_start.rst

```diff
@@ -0,0 +1,191 @@
+=============================
+Getting Started with Packetry
+=============================
+
+Follow these steps to start analyzing live USB traffic with Packetry and Cynthion. If you want to use Packetry only to view existing capture files, you can skip the Cynthion steps.
+
+Install Cynthion
+----------------
+
+Before installing Packetry, follow the instructions to `install Cynthion host software <https://cynthion.readthedocs.io/en/latest/getting_started.html>`__ and then ensure that your Cynthion is up to date by running:
+
+.. code::
+
+   cynthion update
+
+
+Install Packetry
+----------------
+
+.. tab:: Linux
+
+    .. tab:: Binary
+
+        AppImage coming soon! For now you can build and install with Cargo.
+
+    .. tab:: Cargo
```

Comment:
I feel like "Pre-built" and "From source code" would be friendlier headings than "Binary" and "Cargo" - both here and in the other OS sections.